include/swift/Option/Options.td

```diff
@@ -1805,6 +1805,16 @@ def nostartfiles:
          HelpHidden, NewDriverOnlyOption]>,
   HelpText<"Do not link in the Swift language startup routines">;
 
+def enable_cas: Flag<["-"], "enable-cas">,
+  Flags<[FrontendOption, NewDriverOnlyOption]>,
+  HelpText<"Enable CAS for swift-frontend">;
+def cas_path: Separate<["-"], "cas-path">,
+  Flags<[FrontendOption, NewDriverOnlyOption]>,
+  HelpText<"Path to CAS">, MetaVarName<"<path>">;
+def cas_fs: Separate<["-"], "cas-fs">,
+  Flags<[FrontendOption, NewDriverOnlyOption]>,
```

Comment:
Should this be `NoDriverOption`?  I would only expect the frontend to handle this.

Reply:
> Should this be `NoDriverOption`? I would only expect the frontend to handle this.

How do we expect the driver to figure out the argument to this option?

Reply:
NoDriverOptions is reasonable.

Reply:
For clarity if anyone is confused: I must have somehow hit "edit" when I meant to reply to @artemcm above. His post was
> How do we expect the driver to figure out the argument to this option?
and my reply was
> This should be determined by dependency scanning not the driver per se

Sorry!